acceptance/openstack/networking/v2/extensions/qos/policies/policies_test.go

```diff
@@ -16,6 +17,12 @@ func TestPoliciesCRUD(t *testing.T) {
 	client, err := clients.NewNetworkV2Client()
 	th.AssertNoErr(t, err)
 
+	extension, err := extensions.Get(client, "qos").Extract()
```

Comment:
Yes! I like that we check the capability of the cloud to enable tests dynamically. It's a nice pattern to follow.
Much better than what I originally did in https://github.com/gophercloud/gophercloud/pull/2268/commits/22e65202a3f4837701247d4eeaa35e5ddd308262